docs/en/ingest-management/migrate-beats-to-agent.asciidoc

```diff
@@ -0,0 +1,513 @@
+indices.
+
+Note that custom dashboards will show duplicated data until you remove {beats}
+from your hosts.
+
+For more information, see the {ref}/aliases.html[Aliases documentation].
+
+[discrete]
+== Migrate index lifecycle policies
+
+Index lifecycle management (ILM) policies in {es} enable you to manage indices
+according to your performance, resiliency, and retention requirements. To learn
+more about ILM, refer to the
+{ref}/index-lifecycle-management.html[ILM documentation].
+
+ILM is configured by default in {beats} (version 7.0 and later) and in {agent}
+(all versions). To view the index lifecycle policies defined in {es}, go to
+*Management > Index Lifecycle Policies*.
+
+[role="screenshot"]
+image::images/migration-index-lifecycle-policies.png[Screen showing how to add a processor to an integration policy]
+
+If you used ILM with {beats}, you'll see index lifecycle policies like
+*filebeat* and *metricbeat* in the list. After migrating to {agent}, you'll see
+polices named *logs* and *metrics*, which encapsulate the ILM policies for all
+`logs-*` and `metrics-*` index templates.
+
+When you migrate from {beats} to {agent}, you have a couple of options for
+migrating index policy settings:
+
+* *Modify the newly created index lifecycle policies (recommended).* As
+mentioned earlier, ILM is enabled by default when the {agent} is installed.
+Index lifecycle policies are created and added to the index templates for
+data streams created by integrations.
++
+If you have existing index lifecycle policies for {beats}, it's highly
+recommended that you modify the lifecycle policies for {agent} to match your
+previous policy. To do this:
++
+--
+. In {kib}, go to *Stack Management > Index Lifecycle Policies* and search for a
+{beats} policy, for example, *filebeat*. Under *Linked indices*, notice you can
+view indices linked to the policy. Click the policy name to see the settings.
+
+. Click the *logs* policy and, if necessary, edit the settings to match the old
+policy.
+
+. Under *Index Lifecycle Policies*, search for another {beats} policies, for
+example, *metricbeat*.
+
+. Click the *metrics* policy and edit the settings to match the old policy.
+--
++
+Optionally delete the {beats} index lifecycle policies when they are no longer
+used by an index.
+
+* *Keep the {beats} policy and apply it to the index templates created for data
+streams.* To preserve an existing policy, modify it, as needed, and apply it to
+all the index templates created for data streams:
++
+--
+. Under *Index Lifecycle Policies*, find the {beats} policy, for example,
+*filebeat*.
+
+. In the *Actions* column, click the *Add policy to index template* icon.
+
+. Under *Index template*, choose a data stream index template, then add the
+policy.
+
+. Repeat this procedure, as required, to apply the policy to other data stream
+index templates.
+--
+
+//QUESTION: The UI gives you the option of setting the rollover alias. Should
```

Comment:
i would prefer to have any recommendation made in this section to be just agent/fleet based. Mentioning items in other components may become stale quickly.